### ISSUE TITLE
Storage of SEDs and Filters for Template-based codes

### ISSUE THREAD

**Issue by sschmidt23** [LSSTDESC/rail_attic#65]
We need to standardize on how to store both the SEDs and the Filters for multiple template-based photo-z codes.  In implementing bpz_lite (a stripped down version of BPZ available in the [issue/64/BPZ](https://github.com/LSSTDESC/RAIL/tree/issue/64/BPZ) branch) I created a directory: 

`rail/estimation/data` that contains `SED` and `FILTER` subdirectories (as well as an `AB` directory with one size zero file, as this directory is used for temp files for BPZ).  This seems like the most logical place to store such files to me, though I am open to other opinions on whether an alternate storage scheme would be best.

For now I just have the CWWSB templates in the `SED` folder, and the cosmoDC2 ugrizy filter curves in `FILTER`.

**Comment by sschmidt23** [comment]
This will affect #49 and will require a few modifications to Delight and how it fetches the filter and SED files.  Thankfully, it appears that Delight and BPZ use the same format for both SED and FILTER files (F_lambda for the flux, angstroms for wl grid, etc....)

**Comment by johannct** [comment]
we need to be prepared *not* to copy over the filters from project, but rather to have a link in the repo to the throughputs repo. This would mean that the project filters live somewhere else than other filters used for photoz, which is not great..... Perhaps make a soft link? But the problem could be worse : we may have to version the set of filters independently of RAIL..... and we may have other DESC repos around needing the same set of filters that RAIL used.... So shouldn't we think instead of a separate repo for filters?

**Comment by sschmidt23** [comment]
I had a path that I set in the config file until I took it out earlier today in the name of simplicity, but I could easily put it back in.  But, I get nervous about keeping such things straight, and I think this once again ties back in with the question of provenance tracking for all of DESC.  Are any groups beyond PZ using photometry?  If so, they will also have to worry about keeping track of the correct fiducial filter curves for different data releases, and a DESC-wide system seems like a good idea.  It's already a bit of a concern, as Delight's "LSST" filters are slightly different in shape than the filters that were assumed for DC2.
